debian/clearwater-snmpd.prerm

```diff
@@ -54,11 +54,12 @@ case "$1" in
     remove|upgrade|deconfigure)
         # Restore original snmpd.conf 
         [ ! -f /etc/snmp/snmpd.conf.pre-clearwater-snmp ] || mv /etc/snmp/snmpd.conf.pre-clearwater-snmp /etc/snmp/snmpd.conf
-        service snmpd restart || true
 
         # Clean up Monit configuration for snmpd
         rm -f /etc/monit/conf.d/snmpd.monit
         reload clearwater-monit &> /dev/null || true
+
+        service snmpd restart || true
```

Comment:
At this point we've taken away monit's control of snmpd, but not uninstalled snmpd. We can't stop it therefore as nothing will restart it